library/src/main/java/org/mustangproject/ZUGFeRD/TransactionCalculator.java

```diff
@@ -39,8 +39,8 @@ protected BigDecimal getTotalPrepaid() {
 	}
 
 	/***
-	 * the invoice total with VAT, corrected by prepaid amount, allowances and
-	 * charges
+	 * the invoice total with VAT, allowances and
+	 * charges, WITHOUT considering prepaid amount
```

Comment:
This was wrong JavaDoc. The prepaid amount is not considered when calculating the grand total (and this is the expected behavior by other methods).